modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -40,6 +41,18 @@ def show
 
         private
 
+        def bgs_service
+          BGS::Services.new(external_uid: target_veteran.participant_id,
+                            external_key: target_veteran.participant_id)
+        end
```

Comment:
I noticed that bgs_service is also defined in claims_api/v2/application_controller.rb; in v1 claims it's defined in claims_api/v1/forms/base.rb. Wondering if we should standardize where we define external services? Following this pattern I would need to also add local_bgs_service to intent_to_file_controller in order to use it for ITF.